src/Playroom/Frame.tsx

```diff
@@ -1,19 +1,18 @@
-import type React from 'react';
-import type { ReactNode } from 'react';
+import type { ComponentType, ReactNode } from 'react';
 import { useParams } from '../utils/params';
 import CatchErrors from './CatchErrors/CatchErrors';
-// @ts-expect-error
 import RenderCode from './RenderCode/RenderCode';
 
 interface FrameProps {
   themes: Record<string, any>;
-  components: Array<any>;
-  FrameComponent: React.ComponentType<{
+  components: Record<string, ComponentType>;
```

Comment:
Should this be `ComponentType<any>` so it will accept any component? Leaving out the generic argument results in it defaulting to `{}`, so only components that don't take any props would be accepted.